packages/vitest/src/node/pool.ts

```diff
@@ -80,21 +81,39 @@ export function createWorkerPool(ctx: Vitest): WorkerPool {
   const runWithFiles = (name: string): RunWithFiles => {
     return async(files, invalidates) => {
       let id = 0
-      await Promise.all(files.map(async(file) => {
+
+      if (ctx.config.web) {
```

Comment:
maybe it's better to create another function like with `createFakePool`?

Reply:
sure!